### ISSUE TITLE
[Bug] Shortcut for my labels menu: scrolling behavior

### ISSUE THREAD

**Issue by math-GH** [FreshRSS/FreshRSS#7307]
> ... while the share menu seems to make the window move a bit further down so that we see every entry (at least the sole one I have), it would be good to be able to see all the entries on the labels menu too.
> 
> ![Image](https://github.com/user-attachments/assets/358004a4-576a-4557-b46e-f975e69c6718) 

 _Originally posted by @Alwaysin in [#2087](https://github.com/FreshRSS/FreshRSS/issues/2087#issuecomment-2631308371)_

**Comment by math-GH** [comment]
I tried to investigate the root of this behavior.

Following could be the problem:
The labels list will be loaded asynchronously via Ajax. While datas are loading the `main.js` script calculates already the `ScrollTop` to jump to the list. The script ist faster than the loading. So the script scrolls down while calculated with an empty list. Then the list is loaded and will be inserted into the dropdown list. Too late for the `ScrollTop`.

Currently I have no idea how to solve it.

That's why: HELP WANTED. PR welcome

**Comment by aledeg** [comment]
Linked to #2270
